### PR TITLE
Sort Plugins Alphabetically

### PR DESCRIPTION
**Pull request by JasonLG1979**

### CODE REVIEW

File: pithos/plugins/auto_volume_normalization.py [hunk 1]
```diff
@@ -58,7 +58,7 @@ def _volume_warning_dialog(self):
             buttons=Gtk.ButtonsType.OK,
             text=text,
             secondary_text=_(
-                'Please lower the volume before resumming playback,\n'
+                'Please lower the volume before resuming playback,\n'
```
Comment (TingPing) [inline pithos/plugins/auto_volume_normalization.py]: Make it another commit.

Comment (JasonLG1979) [reply pithos/plugins/auto_volume_normalization.py]: Done
